std/experimental/logger/core.d

```diff
@@ -1801,6 +1801,12 @@ unittest
     stdThreadLocalLog = new FileLogger("someFile.log");
```

Comment:
D classic would be to put `scope(exit)` right after stdThreadLocalLog or even before it.Putting scope(exit) at the end looks strange as being `scope(exit)` it makes no difference there.

Also there are some good hints on temporary name in std.file I think.